Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/dto/CaseDTO.java

```diff
@@ -0,0 +1,100 @@
+package com.epam.test_generator.dto;
+
+import com.epam.test_generator.entities.Step;
+import com.epam.test_generator.entities.Suit;
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import java.util.Date;
+import java.util.List;
+
+public class CaseDTO {
+    private Long id;
+
+    private String description;
+
+    @JsonManagedReference
+    private List<Step> steps;
+
+    @JsonBackReference
+    private Suit suit;
```

Comment:
you shouldn't have references to hibernate entities in DTOs. This one should be DTO as well